spec/vehicle_spec.rb

```diff
@@ -33,4 +33,42 @@
       expect(@camaro.electric_vehicle?).to eq(false)
     end
   end
+
+  describe '#registration_date' do
+    it 'can tell you the registration date of an ev' do
+      expect(@cruz.registration_date).to eq(nil)
+    end
+  end
+
+  describe '#add_registration_date' do
+    it 'can add a registration date to a vehicle' do
+      expect(@cruz.add_registration_date(Date.today)).to eq(@cruz.registration_date)
+    end
+  end
+
+  describe '#plate_maker' do
```

Comment:
You do have the test for this here! Ok, no need to also have a test for this method in the Facility spec file.